cmd/zed/query/command.go

```diff
@@ -211,6 +211,19 @@ func ParseSourcesAndInputs(paths, includes Includes) ([]string, ast.Proc, error)
 	return paths, query, nil
 }
 
+func isURLWithKnownScheme(path string, schemes ...string) bool {
```

Comment:
Doesn't this belong in pkg/storage then inputflags would call it?

Reply:
Let's do this later.